addons/mail/static/tests/new/call/call_tests.js

```diff
@@ -104,20 +100,11 @@ QUnit.test("no default rtc after joining a chat conversation", async (assert) =>
 
 QUnit.test("no default rtc after joining a group conversation", async (assert) => {
     const pyEnv = await startServer();
-    const [resPartnerId1, resPartnerId2] = pyEnv["res.partner"].create([
-        {
-            name: "Mario",
-        },
+    const [partnerId_1, partnerId_2] = pyEnv["res.partner"].create([
```

Comment:
mix of camel case and snake case intended here?

Reply:
Yes, the `_1` and `_2` makes the number more visible, so it eases readability